LumpNav/LumpNav.py

```diff
@@ -706,6 +762,45 @@ def setupNavigationPanel(self):
     self.deleteLastFiducialDuringNavigationButton.setEnabled(False)
     self.contourAdjustmentFormLayout.addRow(self.deleteLastFiducialDuringNavigationButton)
 
+    # Distance Collapsible
+    self.distanceCollapsibleButton = ctk.ctkCollapsibleGroupBox()
+    self.distanceCollapsibleButton.title = "Distance"
+    self.distanceCollapsibleButton.collapsed=True
+    self.navigationCollapsibleLayout.addRow(self.distanceCollapsibleButton)
+
+    # Layout within the collapsible button
+    self.distanceFormLayout = qt.QFormLayout(self.distanceCollapsibleButton)        
+
+    self.activateDistanceButton = qt.QPushButton("Activate")
+    setButtonStyle(self.activateDistanceButton)
+    self.activateDistanceButton.setCheckable(True)
+    self.distanceFormLayout.addRow(self.activateDistanceButton)
+    
+    self.distanceComboBoxLabel = qt.QLabel('Text location:')

```

Comment:
There is no time during an intervention to choose an optimal text location: it has to be a fixed setting that is determined by previous experiments; or there is no single setting that works well every time then it would need to be determined automatically (from the viewing angle, target position, etc).

Visibility: Would the surgeons want to change crosshair and trajectory visibility during the procedure? If the information is helpful then it should be probably always displayed, if not helpful (clutters the scene more than it is useful) then it should not be displayed. It is rare that during a procedure you need to change visualization options while in a particular workflow step. If the visibility setting does not neet to be changed during the procedure then it is enough to have a bool flag in the .ini file. If we need to modify it sometimes then the setting should be moved into some advanced section that is hidden by default.